docs/tutorial/node/src/memory_tile.md

```diff
@@ -1,43 +1,52 @@
 <!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+
 # Memory Tile
 
-With the skeleton in place, let's look at the first element of the game, the memory tile. It will be the
-visual building block that consists of an underlying filled rectangle background, the icon image. Later we'll add a
-covering rectangle that acts as a curtain. The background rectangle is declared to be 64 logical pixels wide and tall,
-and it's filled with a soothing tone of blue. Note how lengths in the `.slint` language have a unit, here
-the `px` suffix. That makes the code easier to read and the compiler can detect when your are accidentally
-mixing values with different units attached to them.
+With the skeleton in place, this step looks at the first element of the game, the memory tile. It's the
+visual building block that consists of an underlying filled rectangle background, the icon image. Later you'll add a
+covering rectangle that acts as a curtain.
+
+You declare the background rectangle as 64 logical pixels wide and tall
+and it's filled with a soothing tone of blue.
+
+Note how lengths in the `.slint` language have a unit, here
+the `px` suffix. That makes the code easier to read and the compiler can detect when you accidentally
+mix values with different units attached to them.
 
-We copy the following code into the `memory.slint` file:
+Copy the following into the `ui/appwindow.slint` file:
 
 ```slint
 {{#include memory_tile.slint:main_window}}
 ```
 
-Note that we export the <span class="hljs-title">MainWindow</span> component. This is necessary so that we can later access it
-from our business logic.
+The code exports the <span class="hljs-title">MainWindow</span> component so that the business logic can access it later.
 
-Inside the <span class="hljs-built_in">Rectangle</span> we place an <span class="hljs-built_in">Image</span> element that
-loads an icon with the <span class="hljs-built_in">@image-url()</span> macro. The path is relative to the folder in which
-the `memory.slint` is located. This icon and others we're going to use later need to be installed first. You can download a
-[Zip archive](https://slint.dev/blog/memory-game-tutorial/icons.zip) that we have prepared.
+Inside the <span class="hljs-built_in">Rectangle</span> place an <span class="hljs-built_in">Image</span> element that
+loads an icon with the <span class="hljs-built_in">@image-url()</span> macro.
+
+The path is relative to the folder that contains `ui/appwindow.slint`. You need to install this icon and others you use later first. You can download a pre-prepared
+[Zip archive](https://slint.dev/blog/memory-game-tutorial/icons.zip) and extract it with the
+following two commands:
 
 If you are on Linux or macOS, download and extract it with the following two commands:
 
 ```sh
+cd ui
 curl -O https://slint.dev/blog/memory-game-tutorial/icons.zip
 unzip icons.zip
+cd ..
 ```
 
 If you are on Windows, use the following commands:
 
-```
+```sh
```

Comment:
I think this snippet also needs to account for the ui/ folder. Either by cd or by adjusting the paths.